### ISSUE TITLE
Allign the versions of the jackson jar files to be consistent 

### ISSUE THREAD

**Issue by vhoblik** [oracle/oci-java-sdk#509]
The versions of the jackson jar files are inconsistent

```
com.fasterxml.jackson.core:jackson-annotations:jar:2.13.1
com.fasterxml.jackson.core:jackson-core:jar:2.13.1
com.fasterxml.jackson.core:jackson-databind:jar:2.13.4.2
com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.12.2
```

**Comment by douglaspasqualin** [comment]
Indeed. I'm using the lastest version of jersey3 implementation https://mvnrepository.com/artifact/com.oracle.oci.sdk/oci-java-sdk-common-httpclient-jersey3/3.29.0 . In this version jackson-core is **2.15.2** and jackson-databind **2.13.4.2**.

I tried to update my application to spring-boot 3.2 and during the deploy phase I got the exception: "class com.fasterxml.jackson.databind.cfg.DatatypeFeatures not found". This class was introduced in jackson-databind 2.14 (https://fasterxml.github.io/jackson-databind/javadoc/2.14/com/fasterxml/jackson/databind/cfg/DatatypeFeatures.html).

My current workaround was to force to import an updated version of jackson-databind:

![image](https://github.com/oracle/oci-java-sdk/assets/11462987/652f1592-c72c-4c5a-b139-dce9b16572c2)
